trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -361,8 +369,10 @@ void TrackerNode::control()
   {
     if (path_length < min_track_path_ || std::abs(remain_local) < stop_tolerance_dist_)
     {
-      angle = trajectory_tracker::angleNormalized(-lpath.back().yaw_);
+      angle = trajectory_tracker::angleNormalized(-(it_local_goal - 1)->yaw_);
```

Comment:
this bug had been existed before the refactoring

Reply:
got it. thanks.